samples/force-sensitive-resistor/FsrWithAdcSample.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using Iot.Device.Mcp3008;
+
+namespace force_sensitive_resistor
+{
+    class FsrWithAdcSample
+    {
+        private int _pinNumber = 0; // pin number to which FSR output connected
+        private int _resistance { get; set; } = 10_000; // kOhm
+        private int _voltageSupplied { get; set; } = 3_300;  // 3300mV = 3.3V
+
+
+        private int CalculateVoltage(int readValue)
+        {
+            // This sample used Mcp3008 ADC which analog voltage read output ranges from 0 to 1023 (10 bit) 
+            // mapping it to corresponding milli voltage, update output range if you use different ADC
+            return _voltageSupplied * readValue / 1023;
+        }
+
+        private int CalculateFsrResistance(int fsrVoltage)
+        {
+            // Formula: FSR = ((Vcc - V) * R) / V
+            if (fsrVoltage > 0)
```

Comment:
why do you need to check if > 0? negative voltage is fine too

Reply:
read value would never be negative => voltage would never be negative => if it is negative that means something wrong